cnxarchive/views.py

```diff
@@ -115,7 +115,7 @@ def is_latest(id, version):
 
 
 TYPE_INFO = []
-LEGACY_EXTENSION_MAP = {'epub': 'epub', 'pdf': 'pdf', 'zip': 'complete.zip'}
+LEGACY_EXTENSION_MAP = {'epub': 'epub', 'pdf': 'pdf', 'zip': 'offline.zip'}
```

Comment:
See Karen's comment in #420, which was the basis for this change. But based on your comment, it sounds like this change should be reverted.